src/AzureExtension/QuickStartPlayground/AzureOpenAIService.cs

```diff
@@ -127,24 +142,43 @@ public async Task<string> GetAICompletionAsync(string systemInstructions, string
         var openAIClient = OpenAIClient;
         ArgumentNullException.ThrowIfNull(openAIClient);
 
+        var prompts = systemInstructions + "\n\n" + userMessage;

```

Comment:
Are two new lines required?  If so, can you add a comment that links to the specification?

Reply:
Hm, I don't know if it's _strictly_ required, but this is how we've chosen to format the combined payload of instructions and user prompt. There's no specification for this per se, so I don't know that we have anything here to link to.